src/cljs/org/broadinstitute/firecloud_ui/page/workspaces.cljs

```diff
@@ -129,19 +129,38 @@
   {:render
    (fn [{:keys [state refs]}]
      [:div {}
+      ;; Begin overlay
+      [:div {:style {:visibility (if-not (:overlay-shown? @state) "hidden")
```

Comment:
Any reason for `{:visibility "hidden"}` instead of `{:display "none"}`? I generally prefer the latter unless the former is required.